### ISSUE TITLE
App often shuts down when removing a project

### ISSUE THREAD

**Issue by mooner1022** [mooner1022/StarLight#31]
Stack Trace:
```
*** 치명적인 오류가 발생했습니다. 앱을 종료하는 중... ***
[버그 제보시 아래 메세지를 첨부해주세요.]
──────────
StarLight v0.2.0a-231223(build 2)
PluginCore v0.2.0
Build.VERSION.SDK_INT: 33
Build.DEVICE: redfin
thread  : main
message : Inconsistency detected. Invalid item position 4(offset:-1).state:5 androidx.recyclerview.widget.RecyclerView{15a6525 VFED..... ......ID 0,0-1012,756 #7f0a0253 app:id/recyclerViewProjectList}, adapter:com.mikepenz.fastadapter.FastAdapter@8e8b9f0, layout:androidx.recyclerview.widget.LinearLayoutManager@e932e7d, context:dev.mooner.starlight.MainActivity@fe790f8
cause   : null
┉┉┉┉┉┉┉┉┉┉
Stack Trace:
java.lang.IndexOutOfBoundsException: Inconsistency detected. Invalid item position 4(offset:-1).state:5 androidx.recyclerview.widget.RecyclerView{15a6525 VFED..... ......ID 0,0-1012,756 #7f0a0253 app:id/recyclerViewProjectList}, adapter:com.mikepenz.fastadapter.FastAdapter@8e8b9f0, layout:androidx.recyclerview.widget.LinearLayoutManager@e932e7d, context:dev.mooner.starlight.MainActivity@fe790f8
   at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6821)
   at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6757)
   at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6753)
   at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2362)
   at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1662)
   at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1622)
   at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:687)
   at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep1(RecyclerView.java:4591)
   at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:4346)
   at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4919)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
   at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1891)
   at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1729)
   at android.widget.LinearLayout.onLayout(LinearLayout.java:1638)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
   at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at androidx.recyclerview.widget.RecyclerView$LayoutManager.layoutDecoratedWithMargins(RecyclerView.java:10407)
   at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1720)
   at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1622)
   at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:687)
   at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4645)
   at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:4348)
   at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4919)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at androidx.viewpager2.widget.ViewPager2.onLayout(ViewPager2.java:535)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild(HeaderScrollingViewBehavior.java:149)
   at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:43)
   at com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior.onLayoutChild(AppBarLayout.java:2376)
   at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:918)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at androidx.constraintlayout.widget.ConstraintLayout.onLayout(ConstraintLayout.java:1873)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
   at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1891)
   at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1729)
   at android.widget.LinearLayout.onLayout(LinearLayout.java:1638)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
   at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1891)
   at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1729)
   at android.widget.LinearLayout.onLayout(LinearLayout.java:1638)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
   at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
   at com.android.internal.policy.DecorView.onLayout(DecorView.java:799)
   at android.view.View.layout(View.java:23798)
   at android.view.ViewGroup.layout(ViewGroup.java:6413)
   at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:4118)
   at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3459)
   at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2371)
   at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:9297)
   at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1231)
   at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
   at android.view.Choreographer.doCallbacks(Choreographer.java:899)
   at android.view.Choreographer.doFrame(Choreographer.java:832)
   at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
   at android.os.Handler.handleCallback(Handler.java:942)
   at android.os.Handler.dispatchMessage(Handler.java:99)
   at android.os.Looper.loopOnce(Looper.java:201)
   at android.os.Looper.loop(Looper.java:288)
   at android.app.ActivityThread.main(ActivityThread.java:7918)
   at java.lang.reflect.Method.invoke(Native Method)
   at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)

──────────
```

**Comment by mooner1022** [comment]
Cannot reproduce the error on any of my environments. This issue won't be fixed until I find the condition which triggers the error.

**Comment by mooner1022** [comment]
Fixed in 0.3.0, changes will be included in later commits.
